lite/examples/pose_estimation/android/app/src/main/java/org/tensorflow/lite/examples/poseestimation/ImageUtils.kt

```diff
@@ -0,0 +1,154 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================
+*/
+
+package org.tensorflow.lite.examples.poseestimation
+
+import android.annotation.TargetApi
+import android.graphics.*
+import android.media.Image.Plane
+import android.os.Build.VERSION_CODES
+import android.util.Log
+import androidx.annotation.RequiresApi
+import java.io.ByteArrayOutputStream
+import java.nio.ByteBuffer
+
+/** Utility class for manipulating images.  */
+object ImageUtils {
+
+    private const val TAG = "ImageUtils"
+
+    @RequiresApi(VERSION_CODES.KITKAT)
+    fun yuv420ThreePlanesToNV21(
```

Comment:
Can we use Support.image for image transform?

Reply:
Unfortunately no. YUV images aren't supported yet in the Support Library.
@lu-wang-g to confirm and also review this PR.

Reply:
@lintian06 Checked with Lu offline and YUV images are indeed not supported in Support Library yet. They are only supported in Task Library.